### ISSUE TITLE
Which COCO categories are **not** covered by the pre-trained ssd300 and ssd512 models?

### ISSUE THREAD

**Issue by sachinsdate** [pierluigiferrari/ssd_keras#147]
Hi Pierluigi,
First of all, thanks for creating this truly superb and well documented Keras port!!
A quick question:
Can you please confirm that your pre-trained SSD COCO models are **not** trained on the following COCO categories:

hat
shoe
eye glasses
street sign
plate
mirror
desk
door
window
blender
hair brush

![coco_categories_2017](https://user-images.githubusercontent.com/6522272/42124811-40e889f0-7c87-11e8-8a39-a71ff0f98587.png)

Thanks
 Sachin

**Comment by pierluigiferrari** [comment]
Glad you find it useful!

I'm not sure I understand your question. The COCO models are trained on MS COCO, i.e. on the 80 classes that MS COCO has. That's all there is to say about it, isn't it?

**Comment by sachinsdate** [comment]
Thanks for replying.  I realize now that the categories I mentioned as missing have been removed from MS COCO. They are part of COCO stuff.

See [COCO Labels](https://github.com/nightrome/cocostuff/blob/master/labels.md)

Thank you
